easybuild/easyconfigs/l/LLVM/LLVM-19.1.1-GCCcore-13.3.0-minimal.eb

```diff
@@ -0,0 +1,73 @@
+name = 'LLVM'
+version = '19.1.1'
+versionsuffix = '-minimal'
+
+homepage = "https://llvm.org/"
+description = """The LLVM Core libraries provide a modern source- and target-independent
+ optimizer, along with code generation support for many popular CPUs
+ (as well as some less common ones!) These libraries are built around a well
+ specified code representation known as the LLVM intermediate representation
+ ("LLVM IR"). The LLVM Core libraries are well documented, and it is
+ particularly easy to invent your own language (or port an existing compiler)
+ to use LLVM as an optimizer and code generator."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+toolchainopts = {
+    'pic': True,
+}
```

Comment:
`pic` toolchain option is no longer supported in EasyBuild 5.x when using `system` toolchain, since it wasn't actually doing anything, see also https://github.com/easybuilders/easybuild-framework/pull/4585

```suggestion
```